src/main/java/com/gmail/inverseconduit/commands/sets/CoreBotCommands.java

```diff
@@ -31,6 +31,8 @@
 
     private static final Pattern     javadocPattern = Pattern.compile("^" + Pattern.quote(BOT_CONFIG.getTrigger()) + "javadoc (.*)", Pattern.DOTALL);
 
+    private static final Pattern     TELL_PATTERN   = Pattern.compile("^" + Pattern.quote(BOT_CONFIG.getTrigger()) + "tell (.*?) (.*)$");
```

Comment:
that doesn't actually fix the two spaces problem...

`"tell +([^ ]+) +(.*)"` might be even more robust.